R/metab.bookkeep.R

```diff
@@ -64,7 +64,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
   
 	if("datetime"%in%names(mb.args)){ # check to see if datetime is in the ... args
 		datetime <- mb.args$datetime # extract datetime
-		freq <- calc.freq(datetime) # calculate sampling frequency from datetime
+		freq <- calc.freq(datetime) - 1 # calculate sampling frequency from datetime
```

Comment:
Isn't `calc.freq` returning the same thing as `nobs`? It'll return the number of observations in the day. So I don't see how this could be OK, but the other one (`freq <- nobs -1`) needing the `-1` adjustment.